docs/migrations/web-react/MIGRATION-v3.md

```diff
@@ -86,6 +104,50 @@ Manually replace the props in your project.
 - `<Link isUnderlined … />` → `<Link underlined="always" … />`
 </details>
 
+### Link: Removed `inverted` Variant
+
+The `inverted` variant was removed from the `Link` component.
+Use themes instead to switch the color scheme.
+
+### Pill: Removed Action Colors from `color` Prop
+
+The `color` prop of the `Pill` component no longer supports `primary`, `secondary`, `tertiary`, `inverted`
+and `unselected` variants. Instead, the `neutral` variant was added. The current list of variants is:
+
+- `neutral`
+- `danger`
+- `informative`
+- `success`
+- `warning`
+
+### ProductLogo: Removed `inverted` Variant
+
+The `inverted` variant was removed from the `ProductLogo` component.
+Use themes instead to switch the color scheme.
```

Comment:
I think themes will not help here, `ProductLogo` is now independent on themes. Users may want to switch the actual image source though.